components/event-block/EventBlockSchedule.vue

```diff
@@ -0,0 +1,539 @@
+<template>
+  <div>
+    <div class="event-block-schedule-wide">
+      <div
+        class="event-block-schedule-wide__tablist"
+        role="tablist"
+      >
+        <a
+          v-for="eventDay in parsedEventDays"
+          :key="`tab-wide-${eventDay.id}`"
+          class="event-block-schedule-wide__tablink"
+          role="tab"
+          href="#"
+          :id="`tab-wide-${eventDay.id}`"
+          :aria-controls="`section-wide-${eventDay.id}`"
+          :aria-selected="eventDay.id === activeEventDayId"
+          @click.prevent="selectedEventDayId = eventDay.id"
+        >
+          <time :datetime="eventDay.date">
+            {{
+              new Date(eventDay.parsedDate).toLocaleDateString(
```

Comment:
**suggestion:** use helper method `displayDate(eventDay.parsedDate)` so you can reuse it in mobile template?